io/src/csv/read_measurements.hpp

```diff
@@ -22,6 +22,6 @@ namespace traccc::io::csv {
 /// @param filename The file to read the measurement data from
```

Comment:
Just to give you some actual comment, you should add a line in the documentation about the new parameter. :wink:

Reply:
Sure I often forget doing it  ;p